pkg/cmd/api/spec/metadata.swagger.json

```diff
@@ -0,0 +1,19 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "metadata.proto",
```

Comment:
What is this swagger for?

Reply:
this generated swagger file should be ignored

Reply:
i will fix it

Reply:
done